src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -1483,6 +1531,85 @@ export default Vue.extend({
       'updateDefaultCaptionSettings',
       'showToast',
       'sponsorBlockSkipSegments'
-    ])
+    ]),
+    addPlayerStatsEvent: function() {
+      this.player.on('volumechange', () => {
+        this.stats.volume = this.player.volume()
+        this.player.trigger(this.stats.display.event)
+      })
+
+      this.player.on('timeupdate', () => {
+        const stats = this.player.tech({ IWillNotUseThisInPlugins: true }).vhs.stats
+        this.stats.frameInfo = stats.videoPlaybackQuality
+        this.player.trigger(this.stats.display.event)
+      })
+
+      this.player.on('progress', () => {
+        const stats = this.player.tech({ IWillNotUseThisInPlugins: true }).vhs.stats
+
+        this.stats.bandwidth = stats.bandwidth
+        this.stats.bufferPercent = this.player.bufferedPercent()
+        switch (this.player.networkState()) {
+          case 0:
+            this.stats.networkState = 'NETWORK_EMPTY'
+            break
+          case 1:
+            this.stats.networkState = 'NETWORK_IDLE'
+            break
+          case 2:
+            this.stats.networkState = 'NETWORK_LOADING'
+            break
+          case 3:
+            this.stats.networkState = 'NETWORK_NO_SOURCE'
+            break
+        }
+        this.player.trigger(this.stats.display.event)
+      })
+
+      this.player.on('playerresize', () => {
+        this.stats.playerResolution = this.player.currentDimensions()
+        this.player.trigger(this.stats.display.event)
+      })
+
+      this.createStatsModal()
+      // keyboard shortcut
+      window.addEventListener('keyup', (event) => {
+        if (event.code === 'ArrowUp') {
```

Comment:
I'm not sure if Up Arrow should be used as the keyboard shortcut because I'm not sure if it would negatively effect accessibility @jasonhenriquez can you confirm?

Reply:
do you think a button would be better or another key?

Reply:
I believe other people raised concerns over pages being cluttered with too many buttons so I think just using a different key would be better

Reply:
I've changed the "arrowUp" for "i"